examples/ios/platform_mappings

```diff
@@ -0,0 +1,20 @@
+flags:
```

Comment:
Why are platform mappings needed? Could transitions not be used?

Reply:
the apple rules don't currently support platforms, so the transitions from those rules don't enforce a platform on their dependencies yet

Reply:
Is there a tracking ticket for this somewhere?

Reply:
Was a ticket made but not linked?

Reply:
it's on the first line in this file?

Reply:
Ah! https://github.com/bazelbuild/rules_apple/issues/1658

Thank you! Sorry for the confusion 😅

Reply:
npnp <3